### ISSUE TITLE
Improve feedback when using Wang tiles

### ISSUE THREAD

**Issue by bjorn** [bjorn/tiled#1730]
[Feedback](https://github.com/bjorn/tiled/issues/1455#issuecomment-326263537) from @therahedwig:
 
> When a wang tile transition won't draw because it is missing, could we maybe get some feedback about which one is missing? Like, in the overlay somehow?
>
> ![missing_wang_tile_transition](https://user-images.githubusercontent.com/1759191/29920230-d8332b50-8e4c-11e7-8ab1-c6947f3ce781.png)
Because I am puzzling which one is missing here(it tries to draw grass...)

Maybe @Bdtrotte could comment on the general possibility of this request. I think in general it isn't the case that there is a single Wang ID missing, but maybe this doesn't matter?

**Comment by therahedwig** [comment]
Well, the status bar tells us there is a wang tile missing in this situation.

**Comment by bjorn** [comment]
@therahedwig Yes, but there can be multiple options, depending on the neighboring tiles. But I think it  just render the colors of the first or last one it tried.

**Comment by Bdtrotte** [comment]
@bjorn If I remember correctly, that's not quite how it works: it checks all the surrounding tiles to build a desired wangId, then checks if it exists. You'd then have to go back through and change each corner/edge (probably with wild) and see if that exists. This could end up being non trivial, but I'm not sure...

**Comment by Bdtrotte** [comment]
Pretty scarce for time now, but I'll try to find some time to look at this and the other issues/improvements, and at least pitch in ideas
